lib/resources.js

```diff
@@ -931,18 +945,18 @@ const buildElasticQuery = function (params) {
   // Build ES query:
   var query = {}
 
-  // clean up params
-  ;['q'].forEach(function (param) {
-    if (params[param]) {
-      params[param] = params[param].replace(/date:/g, 'dateStartYear:')
-      params[param] = params[param].replace(/location:/g, 'locations:')
-      params[param] = params[param].replace(/subject:/g, 'subjectLiteral:')
-    }
-  })
+    // clean up params
+    ;['q'].forEach(function (param) {
+      if (params[param]) {
+        params[param] = params[param].replace(/date:/g, 'dateStartYear:')
+        params[param] = params[param].replace(/location:/g, 'locations:')
+        params[param] = params[param].replace(/subject:/g, 'subjectLiteral:')
+      }
+    })
```

Comment:
Did standardjs obligate this change in indentation?

Reply:
yea i don't think it would have ended up here if standard didn't want it.